build_cmake/CMakeLists.txt

```diff
@@ -523,6 +523,25 @@ if (UNIX AND NOT APPLE AND NOT ANDROID)
 
 	add_executable(Play ${QT_SOURCES} ${QT_MOC_SRCS} ${QT_RES_SOURCES} ${QT_UI_HEADERS})
 	target_link_libraries(Play ${PROJECT_LIBS})
+
+	install(TARGETS Play DESTINATION lib/Play/bin)
```

Comment:
looking at this, I think `Play` should be renamed to `Play.bin` and installed to `bin` directory
while script `Play` can either stay as is, or get renamed to `Play.sh`

that should allow the install to be clearer in some sense.